parts/k8s/kubernetesbase.t

```diff
@@ -146,54 +146,7 @@
       "location": "[variables('location')]",
       "name": "[variables('nsgName')]",
       "properties": {
-        "securityRules": [
-{{if .HasWindows}}
```

Comment:
AKS doesn't have Windows, so this never applies

Reply:
Can we indent all the conditional blocks so that it is easier to follow?

Reply:
Done. https://github.com/Azure/acs-engine/pull/4164/files?w=1 to see the diff without whitespaces.